app/javascript/netlify.toml

```diff
@@ -9,3 +9,4 @@
   # base set in Netlify settings UI
   command = "npm install && npm run build-storybook"
   publish = "storybook-static"
+  ignore = "git diff --quiet HEAD^ HEAD app/javascript && git diff --quiet HEAD^ HEAD app/assets/stylesheets/config"
```

Comment:
We need to check for JS changes as well as changes to CSS utility classes which is why we have the custom diff command.